.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐛 Bug Fix
+about: Did you fix something that did not work as expected?
+---
+
+<!---
+Thanks for filing a pull request 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting since someone might have pushed the same thing before!
+-->
+
+# 🐛 bug fix
+
+<!--- Provide the details of the pull request here -->
+
+## 🤔 What was not working?
+
+<!--- Tell us what should happen -->
+
+## 💁 Your Solution
+
+<!--- Describe how you fixed the bug -->
+
+## 🔦 Tests
+
+<!--- Providing tests is not always necessary but helps us test for future bugs -->
```

Comment:
Perhaps change this to test instructions? Mainly in case unit tests aren't sufficient.

Reply:
So we should come up with something like
```
<!--- 
Please provide tests or at least test instructions so we can test for this bug in future releases.
Let us know what we should test to make sure that everything works.
-->
```

or how would you word it?

Reply:
Looks good